_guide/identity-language.md

```diff
@@ -21,6 +21,8 @@ People with disabilities suffer from considerable discrimination, some of which
 
 When talking about people with a lived experience of disability, we want to be respectful of how people want to be identified. This changes over time, and so it is important to stay up-to-date to see that we are using the language that embraces the equality we are trying to achieve.
 
+How can we avoid ableist assumptions and [Inspiration porn](https://en.wikipedia.org/wiki/Inspiration_porn)?
```

Comment:
I understand how the word "porn" is used here (I've heard this term before), and that the ick factor is intentional. But I still feel weird about posting that term on our web site. I wish there were a more professional / less content-warning-ish way to describe it. Disability exploitation? I think we should run this one by Marissa or Marc J to get their thoughts.

Reply:
Ok, I think we can switch over to https://en.wikipedia.org/wiki/Inspiration_Sensationalism